packages/components/psammead-radio-schedule/src/index.stories.jsx

```diff
@@ -59,3 +63,38 @@ stories.add(
     ),
   { notes },
 );
+
+storiesOf('Components|RadioSchedule/StartTime', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
+  .add(
+    'Timestamp',
```

Comment:
```suggestion
    'start time text',
```
Not sure if this story is needed since it isn't much different from the actual Timestamp story. If this is to be kept, I think this should be called something else since it is different as it is styled for this component. Not sure on the name though!

Reply:
Good point, that story was redundant so I removed it and only left the Start time (default) one.